shell/platform/android/android_exports.lst

```diff
@@ -11,6 +11,7 @@
     _binary_icudtl_dat_size;
     
     shorebird_check_for_update;
+    shorebird_current_boot_patch_number;
```

Comment:
I thought next_boot_patch_number was the one the device would boot from? How would that differ from the new api?

Reply:
next_boot_patch_number is "on next boot I will try patch X"
current_boot_patch_number is "I'm currently booted from X"

I presume this is for displaying the current full version info?

Reply:
Okay makes sense. So initially both current and next would return 0? When a new patch is downloaded, current would still be 0 and next would be 1? When the new patch has booted current and next would both be 1?

Reply:
> I presume this is for displaying the current full version info?

Yep! The idea is "You're on version 1.0.0+1 (3), version 1.0.0+1 (5) is available" or similar

Reply:
> Okay makes sense. So initially both current and next would return 0? When a new patch is downloaded, current would still be 0 and next would be 1? When the new patch has booted current and next would both be 1?

That's correct – if no new patch is available, these numbers will be the same.